hugr-model/src/v0/mod.rs

```diff
@@ -664,6 +652,24 @@ pub enum Term<'a> {
     ControlType,
 }
 
+/// An item in a list term.
```

Comment:
Given one of the enum variants here is called `Item` I suspect we should find a different term for the enum itself. `Element`, `Member`, ... or maybe rename the variant to `ListItem::Element` - that might be best?

Reply:
Yes, I thought of `ListComponent` and then indeed ListPart :-), yours are shorter==better

Reply:
There's `ListPart`s and `ExtSetPart`s now.